src/utils/ucc_rcache.h

```diff
@@ -38,4 +38,16 @@ ucc_rcache_get(ucc_rcache_t *rcache, void *address, size_t length,
                                        PROT_READ | PROT_WRITE, NULL, region_p));
 }
 
+/* Version of rcache_get with "arg" parameter. It allows passing additional
+   information from mem_reg callabck. For example, it can be used to signal
+   whether the entry was found in the cache or new registration happened. */
+static inline ucc_status_t
+ucc_rcache_get_arg(ucc_rcache_t *rcache, void *address, size_t length,
+                   void *arg, ucc_rcache_region_t **region_p)
+{
+    return ucs_status_to_ucc_status(ucs_rcache_get(
+                                       rcache, address, length,
+                                       PROT_READ | PROT_WRITE, arg, region_p));
+}
+
```

Comment:
can't we add arg to ucc_rcache_get() now?

Reply:
yes, done